frontend/src/app/ui/modals/modal/modal.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="fixed z-50 inset-0 overflow-y-auto">
+  <div class="flex items-end justify-center min-h-screen pt-4 px-4 pb-20 text-center sm:block sm:p-0">
+
+    <div class="fixed inset-0 transition-opacity">
+      <div class="absolute inset-0 bg-gray-500 opacity-75"></div>
+    </div>
+
+    <span class="hidden sm:inline-block sm:align-middle sm:h-screen"></span>&#8203;
```

Comment:
```suggestion
    <span class="hidden sm:inline-block sm:align-middle sm:h-screen"></span>
```
Ja kann weg, nice find!